src/createServer.js

```diff
@@ -1,3 +1,37 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { handleErrors } = require('./HandleErrors');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const [text, queryString] = req.url.split('?');
+    const params = new URLSearchParams(queryString);
+
+    const textToConvert = text.slice(1);
+    const toCase = params.get('toCase');
```

Comment:
Would be better to move it to separate function, let's do it
```suggestion
    const { textToConvert, toCase } = ...(...);
    
    OR
    
        const [textToConvert, toCase] = ...(...);
```